std/didexchange/invitation/models.go

```diff
@@ -39,3 +39,29 @@ type Invitation struct {
 	Type   string            `json:"@type,omitempty"`
 	Thread *decorator.Thread `json:"~thread,omitempty"`
 }
+
+type OOBInvitation struct {
```

Comment:
What is our strategy towards AFGO? Is there something to reference later? I suppose we don't want to switch there yet. This is more of a question, maybe talk about this tomorrow online.

Reply:
Yes, I think this has to be rethought and rewritten, probably use another struct as the Translate-fn return value, so that we don't lose information in the way. Let's enhance and finalise this later.